speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Mediator.inl

```diff
@@ -192,6 +193,7 @@ void Mediator<DecisionMaker>::DriveTick(tCarElt* p_car, tSituation* p_situation)
 template <typename DecisionMaker>
 void Mediator<DecisionMaker>::RaceStart(tTrack* p_track, void* p_carHandle, void** p_carParmHandle, tSituation* p_situation, Recorder* p_recorder)
 {
+    m_tickCount = 0;
```

Comment:
The Mediator and the DecisionMaker now both have their own tickCount, though their values should always be the same. I think it would be better to make a getter in the DecisionMaker, that the Mediator can call